README.md

```diff
@@ -56,8 +56,8 @@ app.post('/photos/upload', upload.array('photos', 12), function (req, res, next)
   // req.body will contain the text fields, if there were any
 })
 
-const cpUpload = upload.fields([{ name: 'avatar', maxCount: 1 }, { name: 'gallery', maxCount: 8 }])
-app.post('/cool-profile', cpUpload, function (req, res, next) {
+const coolOptions = upload.fields([{ name: 'avatar', maxCount: 1 }, { name: 'gallery', maxCount: 8 }])
```

Comment:
Sorry for not catching this the first time around. But I think that we should change the name to `coolUpload`? Since it's not really "options", it's a complete middleware.

Reply:
Sounds good to me, will quickly fix :3

Reply:
In fact, I'll do you one better -- naming it `uploadMiddleware` makes it more clear that it's middleware